src/cycax/cycad/assembly.py

```diff
@@ -22,11 +23,12 @@ class Assembly:
     def __init__(self, part_no: str):
         self.part_no = part_no
         self.decoder = EngineOpenSCAD()
-        self.assembler = AssemblyOpenSCAD(part_no)
+        self.assembler_openscad = AssemblyOpenSCAD(part_no)
```

Comment:
Instantiate the class only when needed. 
As far as I can see this is a short lived instance of an AssemblyEngine. So it can be created and destroyed (automatic in Python) with in the render method.